conda_build/main_pipbuild.py

```diff
@@ -203,12 +204,21 @@ def get_all_dependencies(package, version):
     cmd1 = "conda create -n _pipbuild_ --yes python pip"
     print(cmd1)
     subprocess.Popen(cmd1.split()).wait()
+
     cmd2 = "%s/bin/pip install %s==%s" % (prefix, package, version)
+    cmd3args = ['%s/bin/python' % prefix, '__tmpfile__.py']
+
+    lsdir = os.listdir(prefix)
+    if not "bin" in lsdir and "Scripts" in lsdir:
```

Comment:
I think this will work, but would it be more straightforward to check if the platform was Windows?

    if sys.platform == "win32":

is what we generally use.